### DISCUSSION TITLE
dropdown in navbar on mobile

### DISCUSSION THREAD

**Discussion by chrisvire** [saadeghi/daisyui#2469] [Q&A] [answered]
I have a dropdown in a navbar. The contents of the dropdown is a grid of name or numbers. Is there a way to have the dropdown content be aligned all the way left on mobile browsers instead of relative to the label? I have been looking at the dropdown-* classes and none of them work for me.

This works great on a desktop browser.

![image](https://github.com/saadeghi/daisyui/assets/517814/3605eb82-ca4b-485f-979d-a0bf255ddda1)

![image](https://github.com/saadeghi/daisyui/assets/517814/de3238d2-ecb4-4125-9395-23c0e3ab060a)

But it goes off the screen on a mobile browser.

![image](https://github.com/saadeghi/daisyui/assets/517814/a854d949-d0c1-425d-9647-1404f536497a)

![image](https://github.com/saadeghi/daisyui/assets/517814/bd850d7d-ab84-44b8-b199-857be2cca7e6)

**Comment by chrisvire** [comment]
Are there tailwndcss classes that I can add to have the dropdown-content positioned at the edge of the browser instead of under the label? What would be the `md:<class>` to have it use the default css for dropdown positioning under the label?

**Comment by saadeghi** [comment, marked as answer]
`.dropdown-content` is relative to the `.dropdown` class because it has `position:relative`.  
You can rewrite that by adding `[position:unset]` class to `dropdown` and then adding `absolute left-0` classes to `dropdown-content` so it sticks to the left:  
https://play.tailwindcss.com/BkKv5b3elU

And if you want that behavior only on mobile, use `max-sm:` prefix for those class names.

Let me know if you have any questions.

**Comment by chrisvire** [comment]
That worked great!  I used `max-sm:start-1.5` so that it worked with LTR and RTL languages and gave a bit of space.
